### PR TITLE
[Feature#800][Plugin] Add Spark Sink Doris

### PR DESCRIPTION
**Pull request by xtr1993**
please refer #800

### CODE REVIEW

**Comment by xtr1993** [comment]
I want mege this code to dev about supporting spark-sink-doris, 
but i dont have write access，please help me open permissions
thanks

File: seatunnel-connectors/plugin-spark-sink-doris/pom.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>seatunnel-connectors</artifactId>
+        <groupId>io.github.interestinglab.seatunnel</groupId>
+        <version>2.0.4</version>
```
Comment (CalvinKirs) [inline seatunnel-connectors/plugin-spark-sink-doris/pom.xml:26]: Some small problem. The groupId should be above the artifactId.

File: seatunnel-connectors/plugin-spark-sink-doris/pom.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpcore</artifactId>
+            <version>4.4.4</version>
+        </dependency>
```
Comment (CalvinKirs) [inline seatunnel-connectors/plugin-spark-sink-doris/pom.xml:58]: The version number should be declared in the root pom

**Review by CalvinKirs — COMMENTED** [review]
Generally looks good,let's wait to submit the docs.

File: seatunnel-connectors/plugin-spark-sink-doris/pom.xml (hunk 2 above)
Comment (xtr1993) [reply seatunnel-connectors/plugin-spark-sink-doris/pom.xml]: yeah，I have update these pom and add apache license in code header

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**